bgpd/bgp_routemap.c

```diff
@@ -4167,10 +4169,12 @@ static const char *parse_evpn_rt_type(const char *num_rt_type)
 
 DEFUN_YANG (match_evpn_route_type,
 	    match_evpn_route_type_cmd,
-	    "match evpn route-type <macip|2|multicast|3|prefix|5>",
+	    "match evpn route-type <1|ead|macip|2|multicast|3|prefix|5>",
```

Comment:
Small nit: I guess this string gets displayed on the screen as part of CLI help. If so can you fix this ?
For other routes, it has route name and then route type ==> macip|2, where as you followed a reverse syntax ==> 1|ead.

Reply:
Ack. Fixed. Though the help string for both numerical and string is same but order is maintained for readability.